### DISCUSSION TITLE
Moving horizon estimation (MHE) with no parameter estimation

### DISCUSSION THREAD

**Discussion by Olusanmi-A** [do-mpc/do-mpc#213] [General]
Thanks for providing a very useful tool. 
Anyone else getting a dimension mismatch error when using MHE without any parameter estimation ?
There seems to be a bug in generating _p_cat and _p_sub matrices.

**Comment by dsdhms** [comment]
I was about to add the same issue today. 

There are two issues, I found out!:

1. Dimension mismatch while using MHE without parameter estimation or state only estimation
2. Error while using MHE without any measurement noise.

I am going to post this as an issue, with example files soon.

**Comment by dsdhms** [comment]
Here's is the link to the issue I've opened: https://github.com/do-mpc/do-mpc/issues/215

Regards,

**Comment by 4flixt** [comment]
Hey guys! I am just back from vacation - so apologies for the late reply. I will investigate this further and post an update soon. Thank you for posting this and for reporting the bug :)

Best,
4flixt
